gateway/src/main/resources/application-prod.yml

```
@@ -62,4 +62,8 @@ jwt:
 eureka:
   client:
     service-url:
-      defaultZone: ${PROD_EUREKA_SERVICE_URL}
\ No newline at end of file
+      defaultZone: ${PROD_EUREKA_SERVICE_URL}
+  instance:
+    hostname: ${EUREKA_INSTANCE_HOSTNAME}
+    prefer-ip-address: false
+    non-secure-port: 80
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)